command/server/config.go

```diff
@@ -967,3 +968,8 @@ func (c *Config) Prune() {
 		c.Telemetry.UnusedKeys = nil
 	}
 }
+
+func (c *Config) found(s, k string) {
```

Comment:
From both instances that used this function, wondering when would `s` and `k` be different?

Reply:
My primary reason for introducing this function is to keep parity with our helper in SharedConfig:

```func (c *SharedConfig) found(s, k string) {
	delete(c.UnusedKeys, s)
	c.FoundKeys = append(c.FoundKeys, k)
}```

There are often cases where we transform the capitalization of keys in configs. For example, `result.found("seal", "Seal")` and `result.found("entropy", "Entropy")`. While we currently aren't doing this for the top level config (only the SharedConfig) I'd like to keep this functionality for future development.